content/vac/sc/g/status/community-contracts-token-import.md

```diff
@@ -26,7 +26,7 @@ gantt
     Production Readiness:
 ```
 
-- status: 57%
+- status: 16%
```

Comment:
While reductions are possible when new issues are being created, I found this jump quite big.
My assumption is that in https://github.com/logos-co/roadmap/pull/40/files#diff-ce5acb9b5acfe2cd345519eff34fe3e18bb036f6bd28b7da37457c376d05fc9fR29 I've changed the wrong milestone.

This likely shouldn't have been 57%  in the last update.